### DISCUSSION TITLE
Add documentation on how to migrate database from sqlite to mysql/postgresql

### DISCUSSION THREAD

**Discussion by Irrational-NX** [grafana/grafana#44285] [General]
Hello everyone, 

Our usage of Grafana has been growing this past few months, and we decided to migrate from sqlite to postgresql (we might want HA later). So I scouted Grafana's documentation a bit... Only to find out that there's no documentation on HOW to move from sqlite to mysql or postgres.
There's this article : https://grafana.com/blog/2020/01/13/how-to-migrate-your-configuration-database/
But it does not cover sqlite to mysql/postgres migration.

There's this article though : https://polyglot.jamie.ly/programming/2019/07/01/grafana-sqlite-to-postgres.html
Which I used with grafana 8.3.2, but I've encountered a few problems along the way (password handling with pgloader, Ubuntu's version of pgloader a bit too old...).
So it works... but I feel like there was too much efforts, and try-and-retry involved 😅 

So... yeah... I'm sure I'm not the only one who struggled to migrate the database, I hope there will be a documentation on that in the future 😃

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
